platform-hub-api/app/models/docker_repo.rb

```diff
@@ -2,7 +2,14 @@ class DockerRepo < ApplicationRecord
 
   NAME_REGEX = /\A[a-z]+[a-z0-9\-_\/]*\z/
 
+  ACCESS_STATUS = {
+    pending: 'pending',
+    active: 'active',
+    removing: 'removing'
```

Comment:
When a user has moved from `active` to `removing` and this has been processed, does the record drop altogether (so they don't linger in a `removed` state)?

Reply:
:) just to clarify further: in upcoming work I'll be actually deleting stuff when responding to the results from the backend integration service.